include/boost/redis/request.hpp

```diff
@@ -67,11 +67,17 @@ class request {
        */
       bool cancel_if_unresponded = true;
 
-      /** @brief If this request has a `HELLO` command and this flag
+      /** @brief (Deprecated) If this request has a `HELLO` command and this flag
        * is `true`, it will be moved to the
        * front of the queue of awaiting requests. This makes it
        * possible to send `HELLO` commands and authenticate before other
        * commands are sent.
+       *
+       * @par Deprecated
+       * This field has been superseded by @ref config::setup.
+       * This setup request will always be run first on connection establishment.
+       * Please use it to run any required setup commands.
+       * This field will be removed in subsequent releases.
        */
       bool hello_with_priority = true;
```

Comment:
I have to ask again: We are deprecating but it won't be removed, just renamed to e.g. has_priority and we will hide it from users?